orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -969,13 +969,24 @@ def _extract_trainer_options(self, state_dict):
         state_dict[_utils.state_dict_trainer_options_key()][D_size] = self.options.distributed.data_parallel_size
         state_dict[_utils.state_dict_trainer_options_key()][H_size] = self.options.distributed.horizontal_parallel_size
 
+    def _extract_train_step_info(self, state_dict):
+        """Extract train step info settings and load it into the state_dict"""
```

Comment:
usually we call it "save to state", load usually refer to restore from state dict

Reply:
Thanks for pointing it out. Will make the changes after merging the file diffs to the original PR.